target/linux/ramips/dts/mt7621_zbtlink_zbt-wg1608.dtsi

```diff
@@ -0,0 +1,211 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+#include <dt-bindings/leds/common.h>
+
+/ {
+	compatible = "zbtlink,zbt-wg1608", "mediatek,mt7621-soc";
+
+	aliases {
+		led-boot = &led_internet;
+		led-failsafe = &led_internet;
+		led-running = &led_internet;
+		led-upgrade = &led_internet;
+		label-mac-device = &gmac0;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			gpios = <&gpio 18 GPIO_ACTIVE_LOW>;
+			linux,code = <KEY_RESTART>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		led_internet: internet {
+			label = "green:internet";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+		};
```

Comment:
Just curious, what is the stock behavior?

Reply:
In stock firmware, it turns on when connected to the internet actually.

Reply:
In stock firmware, it turns on when connected to the internet actually (different from wan led).

Reply:
In stock firmware, it turns on when really connected to the internet. (different from wan led)